apps/stable/cluster-autoscaler/values.yaml.gotmpl

```diff
@@ -0,0 +1,9 @@
+{{- if eq "eks" .Values.jxRequirements.cluster.provider }}
+autoDiscovery:
+  clusterName: {{ .Values.jxRequirements.cluster.provider }}
+awsRegion: {{ .Values.jxRequirements.cluster.region }}
+rbac:
+  create: true
+  serviceAccount:
+    name: cluster-autoscaler
```

Comment:
Doesn't look like the chart is using this. Let's skip `rbac.serviceAccouny.name`